spec/controllers/topics_spec.rb

```diff
@@ -40,7 +40,7 @@
       let(:data) {{
         type: 'create',
         url:  'https://example.com/widgets/123',
-        timestamp: Time.now.to_f
+        timestamp: Time.now.to_i * 1e3
```

Comment:
All timestamps in Routemaster are integer miliseconds - in this test it happened to be a float (for no reason), without failing the test.
Oj is a bit more picky so it exposed the issue with the test.